canonical-web-and-design/jaas.ai
# Update Python testing infrastructure

## Done
- `./run test` run `lint-python` and `test-python`
- `./run lint-python` run `flake8` and `black --check --line-length 79` against all Python code (including tests)
- `./run test-python` run the `tests`
- Set up CircleCI with a job called `test-python`, which run the tests
- Added codecov integration on CircleCI

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8029
- Run the command `./run test`, it should end correctly.
- Run the command `./run lint-python`, it should end correctly.
- Run the command `./run test-python`, it should end correctly.
